### DISCUSSION TITLE
AH/Teensy/TeensyUSBTypes.hpp: No such file or directory

### DISCUSSION THREAD

**Discussion by mattibatti** [tttapa/Control-Surface#1023] [General]
I have been working on a controller with Arduino Micro and I tried to switch my project over to a Teensy. But it turned out Teensy is not compatible with my OS. So I went back to the Micro, but am now getting this error. Any advice?

""In file included from /Users/mattiweisberg/Documents/Arduino/libraries/Control-Surface-1.2.0/src/MIDI_Interfaces/DebugMIDI_Interface.hpp:3:0,
                 from /Users/mattiweisberg/Documents/Arduino/libraries/Control-Surface-1.2.0/src/Control_Surface.h:119,
                 from /Users/mattiweisberg/Documents/Arduino/TEU_Main_2/TEU_Main_2.ino:1:
/Users/mattiweisberg/Documents/Arduino/libraries/Control-Surface-1.2.0/src/MIDI_Interfaces/SerialMIDI_Interface.hpp:6:10: fatal error: AH/Teensy/TeensyUSBTypes.hpp: No such file or directory
 #include <AH/Teensy/TeensyUSBTypes.hpp>"""
          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.

**Comment by tttapa** [comment]
This is the first time I'm seeing this error. I'd recommend reinstalling Control Surface, and while you're at it, I'd recommend installing the latest `main` version.

Which OS are you using that's not compatible with Teensy? Which board do you have selected in the IDE?

**Reply by mattibatti** [reply]
Ok I tried that and it worked for a bit, but now the board stops being recognized as a port after uploading my program. If I restart it, it becomes recognizable again and can receive other programs. But mine makes it disappear. Not sure what to do about that!

And - I have macOS High Sierra (10.13) and tried to connect to a Teensy 4.0

Thanks!
